wpeverest/everest-forms
# Enhance - Add ehnanced select feature to the Dropdown field

### All Submissions:






### Changes proposed in this Pull Request:
This PR adds the `select2` feature to the `Dropdown` field. It is a pro feature.

### How to test the changes in this Pull Request:

1. Activate the `Everest Forms Pro` with a valid license
2. Create a form and add a `Dropdown` field
3. In the field options you'll find `Enable Select2` option. Check the option.
4. Go to the frontend and the `Dropdown` field should have a search box when you click on it

### Types of changes:




### Other information:




### Changelog entry

> Enhance - Add ehnanced select feature to the `Dropdown` field
